src/translate.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ *  gd-tools - a set of programs to enhance goldendict for immersion learning.
+ *  Copyright (C) 2025 Ajatt-Tools
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "precompiled.h"
+#include "subprocess.hpp"
```

Comment:
There's no need to add this file to the project. Just include it as a dependency using xmake.